metadata.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "crayfishx/rsyslog",
+  "name": "puppet-rsyslog",
   "version": "1.1.0",
   "author": "crayfishx",
```

Comment:
please change that to Vox Pupuli

Reply:
module name has to be `puppet-rsyslog` since puppet is our forge namespace, no?

Reply:
Or do you mean author?